robotoff/off.py

```diff
@@ -500,3 +500,25 @@ def select_rotate_image(
 
     r.raise_for_status()
     return r
+
+
+def normalizing(line, lowercase=True):
```

Comment:
I would be more explicit, I would call it `normalize_tag` (it's about tag)

And think about adding documentation !

Also you should rename "line" parameter to "value" maybe ?

```suggestion
def normalize_tag(line, lowercase=True):
    """given a value normalize it to a tag (as in taxonomies)
    
    This means removing accents, lowercasing, replacing spaces with dashes, etc..
    """
```